javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -743,47 +743,122 @@ i18n.phonenumbers.PhoneNumberUtil.VALID_PHONE_NUMBER_ =
 i18n.phonenumbers.PhoneNumberUtil.DEFAULT_EXTN_PREFIX_ = ' ext. ';
 
 
-/**
- * Pattern to capture digits used in an extension.
- * Places a maximum length of '7' for an extension.
+/**  Helper method for constructing regular expressions for parsing. Creates
+ *   an expression that captures up to max_length digits.
  *
- * @const
- * @type {string}
- * @private
+ *   @return {string} constructued RegEx pattern to capture extension digits.
```

Comment:
remove "constructued" (typo anyway and doesn't add clarity)

Reply:
Done.